code/workspaces/auth-service/src/userManagement/authZeroUserManagement.spec.js

```diff
@@ -15,6 +16,9 @@ const cacheReturnedMock = jest.fn();
 const cacheWrapperMock = jest.fn().mockReturnValue(cacheReturnedMock);
 cache.getOrSetCacheAsyncWrapper = cacheWrapperMock;
 
+config.set('userManagementClientId', 'userMgmtClientId');
```

Comment:
Was this required to fix the problem that the tests didn't work when you've got a .envrc file?

Reply:
Yes, have given more detail for the previous comment.